app/javascript/app/components/data-explorer-content/data-explorer-content-selectors.js

```diff
@@ -138,6 +154,25 @@ export const getLink = createSelector(
   }
 );
 
+function getOptions(section, filter, filtersMeta) {
+  if (section !== 'emission-pathways') return filtersMeta[filter];
+  if (filter === 'categories' || filter === 'subcategories') {
+    return filtersMeta.categories;
+  }
+  return filtersMeta[filter];
+}
+
+function parseOptions(section, filter, options) {
+  if (section !== 'emission-pathways') return options;
```

Comment:
use `switch` here maybe?

Reply:
Well I think this is more related with preferences (as in this small statement a `switch` would not add any performance improvement). 
I use to go into `switch`es when there are more cases,  but if you think it could increase redeability I'll go for it

Reply:
I meant for readibility reasons, yes. i think having too many ifs is a bad practice, is all

Reply:
⚡️ done

Reply:
🙏 thank you!